<?xml version="1.0"?>
<launch>


	<arg name="y_topic" 	        default="virtual_classifier"/>
 	<arg name="fixed_frame" 	default="/world"/>
	<arg name="path_sensor_model"   default="$(find objects)/meshes/plug/config/X.txt"/>
	<arg name="peg_link_name"       default="lwr_peg_link"/>
	<arg name="socket_type"         default="one"/>	

	<node name="virtual_peg_sensor" pkg="peg_sensor" type="virtual_peg_sensor_node" args="-y_topic $(arg y_topic) -fixed_frame $(arg fixed_frame) -path_sensor_model $(arg path_sensor_model) -peg_link_name $(arg peg_link_name) -socket_type $(arg socket_type) -rate 100" output="screen"/>


</launch>

